frontend/src/metabase/actions/containers/ActionCreatorModal/ActionCreatorModal.unit.spec.tsx

```diff
@@ -81,18 +87,25 @@ describe("actions > containers > ActionCreatorModal", () => {
   it("renders correctly", async () => {
     const initialRoute = `/model/${MODEL.id}/detail/actions/${ACTION.id}`;
     await setup({ initialRoute });
-    expect(
-      await screen.findByTestId("mock-action-creator"),
-    ).toBeInTheDocument();
+
+    await waitFor(() => {
+      expect(screen.getByTestId("action-creator")).toBeInTheDocument();
+    });
+
+    expect(await screen.findByTestId("action-creator")).toBeInTheDocument();
```

Comment:
I'm not sure what these 2 assertions are supposed to represent. Both are checking the presence of the element with `data-testid=action-creator`. Can we just have one but not both?

Reply:
Yes! Good catch.

7addd6e6e7ea43d79ba93cde187fdcdccda3f973